docs/docsource/installation/install/installing_xdm.rst

```diff
@@ -0,0 +1,517 @@
+          provider: InCluster
+          ui:
+            ingress:
+              hostname: ui.zenko.local
+          oidc:
+            provider: 'http://keycloak.zenko.local/auth/realms/zenko-realm'
+            uiClientId: zenko-ui
+            vaultClientId: zenko-ui
+          api:
+            ingress:
+              hostname: management.zenko.local
+            allowFrom:
+            - 172.16.0.0/12
+            - 10.0.0.0/8
+        ingress:
+          workloadPlaneClass: 'nginx'
+          controlPlaneClass: 'nginx'
+          annotations:
+            nginx.ingress.kubernetes.io/proxy-body-size: 0m
+      EOF
+      kubectl wait --for condition=Available --timeout 10m -n zenko zenko/      
+      zenko-instance
+
+#. Create a Keycloak user for the XDM instance with:
+
+   .. code::
+      
+      OIDC_USER="zenko-tester"
+      INSTANCE_ID=$(kubectl -n zenko get zenko/zenko-instance -o jsonpath='{.      status.instanceID}')
+      REALM_NAME="zenko-realm"
+      
+      kubectl -n ringx-auth exec -i keycloak-0 -- /opt/jboss/keycloak/bin/kcadm.sh       config credentials --server http://localhost:8080/auth --realm master --user       admin --password password
+      cat <<EOF | kubectl -n ringx-auth exec -i keycloak-0 -- /opt/jboss/keycloak/      bin/kcadm.sh create users -r "${REALM_NAME}" -f -   
+      {
+          "username": "${OIDC_USER}",
+          "enabled": true,
+          "totp": false,
+          "emailVerified": true,
+          "firstName": "zenko",
+          "lastName": "tester",
+          "email": "${OIDC_USER}@zenko.local",
+          "attributes": {
+              "instanceIds": [
+                  "${INSTANCE_ID}"
+              ],
+              "role": [
+              "user"
+              ]
+          },
+          "credentials": [],
+          "disableableCredentialTypes": [],
+          "requiredActions": [],
+          "realmRoles": [
+              "uma_authorization",
+              "offline_access"
+          ],
+          "clientRoles": {
+              "account": [
+              "view-profile",
+              "manage-account"
+              ]
+          },
+          "notBefore": 0,
+          "groups": []
+      }
+      EOF
+      
+      kubectl -n ringx-auth exec -i keycloak-0 -- /opt/jboss/keycloak/bin/kcadm.sh       set-password -r ${REALM_NAME} --username ${OIDC_USER}
+
+Testing
+-------
+
+Via the User Interface
+**********************
+
+Add the following hosts to the browser user machine ``/etc/hosts`` to resolve to 
+MetalK8s node's IP address:
+
+- management.zenko.local
+
+- keycloak.zenok.local
+
+- ui.zenko.local
+
+- s3.zenko.local
+
+.. note::
+
+   If the machine has port 80 open, you can access the User Interface through http://ui.zenko.local.
+
+.. important::
+
+   The User Interface is limited to creating accounts and locations.
+
+Via Command Line
+****************
+
+Prerequisites
+~~~~~~~~~~~~~~
+
+- jq and curl must be installed
+- Access from the browser user machine
```

Comment:
You must have browser access to the host

but... if you're doing this from the command line, why do you need browser access?

Reply:
I'm not sure what that means, I asked @terrydervaux